java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/jdbc/PostgresDialect.java

```diff
@@ -27,6 +29,26 @@
 
 public class PostgresDialect implements JdbcDialect {
 
+    private static final HashMap<TypeName, String> RW_TYPE_TO_JDBC_TYPE_NAME;
+
+    static {
+        RW_TYPE_TO_JDBC_TYPE_NAME = new HashMap<TypeName, String>();
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.INT16, "int2");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.INT32, "int4");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.INT64, "int8");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.FLOAT, "float4");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.DOUBLE, "double precision");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.BOOLEAN, "bool");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.VARCHAR, "varchar");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.DECIMAL, "numeric");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.TIME, "time");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.TIMESTAMP, "timestamp");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.INTERVAL, "varchar");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.DATE, "date");
+        RW_TYPE_TO_JDBC_TYPE_NAME.put(TypeName.TIMESTAMPTZ, "timestampz");
```

Comment:
`timestampz` should be `timestamptz`, `tz` in the end

Reply:
Good catch. Fixed in the latest commit.